Gruntfile.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Main entrypoint for grunt.
+ */
+'use strict';
+
+module.exports = function(grunt) {
```

Comment:
If you want to get this out the door, use Grunt for now, but, similar to above, we might want to use webpack here to keep things consistent moving forward.